readthedocs-theme/templates/readthedocs/product.html

```diff
@@ -173,7 +173,7 @@ <h4>Multiple versions</h4>
           </div>
           <div class="extra content">
             <div class="ui labels">
-              <span class="ui label basic large"><i class="icon primary hourglass outline"></i> Version control</span>
+              <span class="ui label basic large"><i class="icon primary copy outline"></i> Version control</span>
```

Comment:
Perhaps:

```suggestion
              <span class="ui label basic large"><i class="icon primary code branch"></i> Version control</span>
```

Reply:
No, the icon I'm describing should be:
![image](https://user-images.githubusercontent.com/1140183/154566582-41ed557b-77d1-4375-801f-0e29b2d8e51b.png)

I get the correct icon that I'm describing:
![image](https://user-images.githubusercontent.com/1140183/154566659-abef26c1-ecaf-492c-8a9d-0e8756824c08.png)

Reply:
Aha, that one is perfect! It's exactly the one I was looking for. I was looking on the Semantic UI docs thought. Glad to see that Fomantic updated it. Makes a lot more sense than the SUI one.

Reply:
Odd, not sure where you were seeing that, I see the same on on SUI's docs:

![image](https://user-images.githubusercontent.com/1140183/154737000-64ee28db-86e1-433a-835f-167c83fd6a0b.png)

Reply:
I was using these docs: https://semantic-ui.com/elements/icon.html#code.
Now changed to the Fomantic equivalent after realising this.

Reply:
ah yeah, so that grid view is wrong. the search shows the correct icon.